.nuke/build.schema.json

```diff
@@ -75,8 +75,6 @@
               "CalculateVersion",
               "Clean",
               "Compile",
-              "CopyToLocalPackages",
```

Comment:
This file is updated from running the Nuke build command locally. The build steps were removed prior to this PR.